parlai/tasks/quac/build.py

```diff
@@ -10,10 +10,18 @@
 import json
 
 VERSION = '0.2'
-TRAIN_FILENAME = 'train_v' + VERSION + '.json'
-VALID_FILENAME = 'val_v' + VERSION + '.json'
+
+FILE_NAMES = ['train_v' + VERSION + '.json', 'val_v' + VERSION + '.json']
 
 URL = 'https://s3.amazonaws.com/my89public/quac/'
+
+URLS = list(map(lambda x: URL + x, FILE_NAMES))
```

Comment:
nit that's too early: you should avoid lambdas pretty much always, and use list comprehensions instead. `URLS = [URL + fn for fn in FILE_NAMES]`